### DISCUSSION TITLE
How can I export my notes?

### DISCUSSION THREAD

**Discussion by hamoid** [quillpad/quillpad#290] [Q&A] [answered]
Hi! I really like Quillpad. It works great for note editing. But I need sync!

Currently the export function doesn't work for me: https://github.com/quillpad/quillpad/issues/266

And the Nextcloud sync fails silently: https://github.com/quillpad/quillpad/issues/259

Any suggestions on how I can get my 30 long notes out of my phone? 

Even if it needs building the program on IntelliJ Idea? (I have it installed for Kotlin development, not Android).

**Comment by tasiaiso** [comment]
For now, you can
- export them to markdown files using markor or another text editor
- uninstall the app and reinstall 1.4.13
- import the last healthy backup
- reimport all of the other text files.

**Reply by hamoid** [reply]
Thank you, but I think I need more details. How do I export the files to markdown? As I mentioned, the feature is not working for me due to issue #266 The app closes when I hit "export".

Do you mean I should install Markor to export the files?

Mmm... of course I could go one by one doing copy and paste, which would be very tedious...

**Reply by hamoid** [reply]
Now I see that maybe Version 1.4.15 solves this issue... but would I loose my notes if I install the apk?

**Reply by jsixface** [reply]
Why do you think you would loose your notes?

**Reply by tasiaiso** [reply]
By "export" i did mean copy-pasting your files over. You can just update it, like any other update ?

**Reply by hamoid** [reply]
@jsixface Over the years I have encountered hundreds of messages in all kinds of software stating I should back up things before updating. I do not know if I would loose my notes or not, therefore I'm asking. A reason to loose the notes might be that switching from an app coming from the store is not considered the same app coming from an apk downloaded from a  website. Since I don't know if this is possible, I thought asking is a good idea.

**Reply by hamoid** [reply]
@tasiaiso Ok... if no better option I will copy paste everything. I could just update to 1.4.15 if it was available in Neo Store, but it's not yet.

**Reply by jsixface** [reply, marked as answer]
I see. I missed the part about downloading the apks. 

It should be available in the play store by now.

**Reply by hamoid** [reply]
Thank you! I'll try it as soon as it shows up in f-droid :)

**Reply by hamoid** [reply]
Ah got it via Aurora store and backup now works! Yay! :-)

**Reply by clach04** [reply]
Only just saw this, I have links to tools in https://github.com/clach04/quillpad_conversion I've not yet tried to use https://github.com/MolassesLover/jotbook but it claims to support converting backup files to text.
